cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -147,7 +147,127 @@ property replaces the ``schedule`` property of an ETL Batch Application.
 
 In the example code above, we will use a *ProjectionTransform* (a type of Transform) to drop certain
 columns in the incoming data. A *StreamSink* in the final step needs a data field property
-that it will use as the content for the data to be written. 
+that it will use as the content for the data to be written.
+
+DAG in transform
+----------------
+
+ETLConfig support DAG in pipeline, which allows non-linear execution of pipeline stages.
```

Comment:
What is ETLConfig? It is not mentioned elsewhere (except as a variable) in the documentation. Did you mean:

```
ETL Applications support directed acyclic graphs in pipelines, which allows for the 
non-linear execution of pipeline stages.